backend/src/test/java/gov/cdc/usds/simplereport/service/OktaServiceTest.java

```diff
@@ -47,6 +48,7 @@
         return USERNAMES;
     }
 
+    @Disabled
```

Comment:
We should create an ticket to re-enable these (or possibly make a note that re-enabling them is a requirement on the ticket I created already on a related topic).

Reply:
Added a note to #653